library/ui-styles/src/main/res/values/styles_edit_text.xml

```diff
@@ -11,4 +11,14 @@
         <item name="android:textColor">?vctr_message_text_color</item>
     </style>
 
-</resources>
\ No newline at end of file
+    <style name="Widget.Vector.EditText.Composer.RichText" parent="Widget.AppCompat.EditText">
```

Comment:
I would rename to `Widget.Vector.EditText.RichTextComposer`, to avoid the confusion with the parenting (parent style is `Widget.AppCompat.EditText`, not `Widget.Vector.EditText.Composer`).

Or eventually just write:

```xml
    <style name="Widget.Vector.EditText.Composer.RichText">
        <item name="android:minHeight">20dp</item>
        <item name="android:padding">0dp</item>
    </style>
```